### DISCUSSION TITLE
Sketchware rc01-kt Kotlin library  problem

### DISCUSSION THREAD

**Discussion by AndroXStudiO** [Sketchware-Pro/Sketchware-Pro#482] [Ideas]
Sketchware rc01-kt. Problem with Kotlin library. 
I have downloaded a library that is made with Kotlin language.

After implement it sketchware rc01-kt build apk successfully but after app open it Crash.

Next time,  i added kotlin JVM library then apk working fine....please fix this issue and make Kotlin JVM library as default for sketchware

**Comment by j-px** [comment]
I don't think that's a great idea. That'd add an unnecessary dependency to projects not using Kotlin.

**Reply by AndroXStudiO** [reply]
make Kotlin JVM library as default for sketchware-kt  (Kotlin Version)

**Reply by j-px** [reply]
It already gets automatically added to the project if it's using Kotlin code: [Click here to view the code adding it automatically](https://github.com/Sketchware-Pro/Sketchware-Pro/blob/2b22943aebd7dbe2734b6c87a1eb6e54913e7135/app/src/minApi26/java/mod/hey/studios/compiler/kotlin/KotlinCompilerBridge.java#L39).
